modules/lang-expression/src/main/java/org/elasticsearch/script/expression/ExpressionScriptEngineService.java

```diff
@@ -79,7 +79,7 @@ public Object compile(String scriptName, String scriptSource, Map<String, String
         // classloader created here
         final SecurityManager sm = System.getSecurityManager();
         if (sm != null) {
-            sm.checkPermission(new SpecialPermission());
+            sm.checkPermission(SpecialPermission.INSTANCE);
```

Comment:
Yes. It is slightly different in form which is why autorefactoring probably did not catch it.